include/rocksdb/utilities/object_registry.h

```diff
@@ -189,17 +290,25 @@ class ObjectRegistry {
     }
   }
 
-  // Dump the contents of the registry to the logger
-  void Dump(Logger* logger) const;
+  // Returns the number of registered types for this registry.
+  // If specified (not-null), types is updated to include the names of the
+  // registered types.
+  virtual size_t GetRegisteredTypes(
+      std::unordered_set<std::string>* types) const = 0;
+  // Returns the number of registered names for the input type
+  // If specified (not-null), names is updated to include the names for the type
+  virtual size_t GetRegisteredNames(const std::string& type) const = 0;
```

Comment:
It is confusion that the function name is "GetRegisteredNames" but returns the number of names of certain type.

Reply:
Would "GetNamesForType" be better?

Reply:
Yes, let's use "GetNamesForType"